packages/opds-web-client/src/components/BookDetails.tsx

```diff
@@ -3,7 +3,7 @@ import * as moment from "moment";
 import BookCover from "./BookCover";
 import Book, { BookProps } from "./Book";
 
-export interface BookDetailsProps extends BookProps {};
```

Comment:
So I'm guessing this is now unnecessary Typescript?

Reply:
I'm actually not sure. The only change I made here was to remove the semicolon because the linter complained when I tried to run the tests. Doesn't relate to my PR though, so I'm not sure how the errant semicolon got through in the first place